addons/binding/org.openhab.binding.nest/ESH-INF/thing/bridge.xml

```diff
@@ -4,46 +4,8 @@
 	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
 
 	<bridge-type id="account">
-		<label>Nest Account</label>
-		<description>An account for the Nest web service.</description>
-		<config-description>
-			<parameter-group name="nest">
-				<description>The pieces for talking to nest</description>
-				<label>Nest Connection</label>
-			</parameter-group>
-			<parameter-group name="binding">
-				<description>Local settings</description>
-				<label>Binding settings</label>
-			</parameter-group>
-
-			<parameter name="clientId" type="text" groupName="nest">
-				<label>Client id</label>
-				<description>The client id from the nest product page</description>
-				<required>true</required>
-			</parameter>
-			<parameter name="clientSecret" type="text" groupName="nest">
-				<label>Client secret</label>
-				<description>The client secret from the nest product page</description>
-				<required>true</required>
-			</parameter>
-			<parameter name="pincode" type="text" groupName="nest">
-				<label>Pincode</label>
-				<description>The pincode to use after clicking on the authorization url</description>
-				<required>true</required>
-			</parameter>
-			<parameter name="accessToken" type="text" groupName="nest">
-				<label>Access Token</label>
-				<description>The access token to use from nest.  Only need to set this value if you want to override
-				the value that is automatically retrieved from nest.  The automatic value is stored elsewhere.</description>
-				<required>false</required>
-				<advanced>true</advanced>
-			</parameter>
-			<parameter name="refreshInterval" type="integer" groupName="binding">
-				<label>Refresh Interval</label>
-				<description>How often to refresh the data from nest (in seconds). Don't make this too often or nest will throttle
-				your account. The default of 2 minutes is roughly the most often you can do it.</description>
-				<default>120</default>
-			</parameter>
-		</config-description>
+		<label>Nest Product Account</label>
```

Comment:
Do you have one account per product and not one nest account for all your products?

Reply:
You have one Nest account for all your Products. Maybe it is better to just name it a "Nest Product" and make it a "product" Thing? The Nest Account to which the Product belongs is actually not used within the binding.

Also, once the Access Token has been obtained the Product ID and Product Secret are no longer used. Unless you want to obtain a new token. So we could also make the Product ID and Product Secret parameters optional. That would also allow for a file based configuration with just the Access Token.

Reply:
Sorry, I cannot follow you here. We have references to the "account" everywhere here and everything around OAuth and the general access to ANY product is going through this account, doesn't it? What I hear is that the account is almost irrelevant and we only use a general REST API - but the authentication is done through OAuth and if I am not mistaken, there should be mechanisms like refresh tokens etc. which are linked to the account - and if we one day support OAuth flows, we might really want an account as a bridge, don't we?

Reply:
What I mean with Nest Account is the combination of e-mail/password that are required to login. 
You create the Products using your Nest Account but the email/password to create these are not used in the binding. In the Nest developer console the products are referred to as "Products" but not "Nest Product Account" or "Nest Account". 

The Nest API currently has no mechanism to refresh tokens (they are valid for 10 years). Most likely the Product ID and Product Secret will be required for refreshing them so it is best if they are already there. The bridge should certainly be there so it is clear which Things belong to what Product.

Reply:
So what I mean is to rename the Bridge from "Nest Product Account" to "Nest Product". :-)

Reply:
> What I mean with Nest Account is the combination of e-mail/password that are required to login.

That's a wrong association, because an account can have many other authorization mechanisms than credentials. But you need to have access to your "account" (i.e. have some authentication details) in order to use Nest.

> So what I mean is to rename the Bridge from "Nest Product Account" to "Nest Product". :-)

So there is one of these bridges for every product that you own? If this is the case, I agree. If there is a single entity, it should stay "Nest Account" as it had been so far.

Reply:
In Nest language the Product is the integration that a developer makes with their API. [Here is a page](https://developers.nest.com/documentation/cloud/edit-client) explaining how Products work. It also has some screenshots of the developer console that people using the Nest binding  have to use to setup a Product for the binding.

Reply:
Normally the Product ID/Secret will be part of the application and all an end user has to do is grant permissions to the application via the Authorization URL. In openHAB there is more freedom and because of that users have to setup their own Product and also enter the Product ID/Secret.

To simplify the binding authorization process, openHAB could also create its own Product and let the binding use this when no Product ID and Product Secret are provided. Users could simply browse to a predefined URL grant permissions and use the Pincode with the binding.

In that case the whole Product aspect is unknown to the user and the bridge would also simply be a "Nest Account". So I'll rename the "Nest Product Account" to "Nest Product".

Reply:
Rename it to a "Nest Account", it is still early :-)